utils.sh

```diff
@@ -47,6 +47,16 @@ function create_cluster() {
     custom_ntp
     bmo_config_map
 
+    # We're generating a config to allow the master nodes to download
+    # from our local container cache, put the same change on the bootstrap node
+    if [ -e ./assets/generated/99_local-registry.yaml ] ; then
+        cp ${assets_dir}/install-config.yaml{.tmp,}
+        $OPENSHIFT_INSTALLER --dir "${assets_dir}" --log-level=debug create ignition-configs
+        yq .spec.config.storage.files[0] ./assets/generated/99_local-registry.yaml | \
```

Comment:
Where is it specified that yq is a requirement to run dev-scripts?

Reply:
It's installed along with the other dependencies here:

https://github.com/metal3-io/metal3-dev-env/blob/master/vm-setup/roles/packages_installation/defaults/main.yml#L117

It'd be nice to remove that requirement since it's a pip install,  there are other approaches we could use such as https://github.com/ashcrow/filetranspiler but for now yq works.